doc/api/cli.md

```diff
@@ -119,6 +119,18 @@ added: v6.0.0
 Force FIPS-compliant crypto on startup. (Cannot be disabled from script code.)
 (Same requirements as `--enable-fips`.)
 
+### `--http-parser=library`
+<!-- YAML
+added: REPLACEME
+-->
+
```

Comment:
Or maybe we should mark this flag itself experimental?

Reply:
Do you see any kind of actual breaking changes that we could make to it? I wouldn’t consider making it a no-op a breaking change. I’d be happy to add a warning about that (possibly no-op-ing it) here, though?

Reply:
My reasoning is that the part being experimental is "the ability to choose your HTTPParser". So, while this flag is experimental, the user cannot always expect that they have the ability to choose that, and the flag can be a noop in a particular release (without going semver-major) because of security reasons, which I think doesn't matter that much for people who do experiment with it, and they will be given proper warning about this fact.

 When we are confident enough we will just remove the old parser along with this flag, or make it a noop. Since in our stability contract, being experimental means the feature itself ("the ability to choose your HTTPParser") may be removed completely.

Reply:
That sounds a bit complex, imo? Wouldn't something "This flag is likely to become a no-op and later be removed at some point in the future" be explicit enough + do the trick?

Reply:
@addaleax What's the difference between that and making this experimental? In my understanding, being experimental means: a flag + a warning when it's used + a orange banner in the documentation + freedom to break it (including making it a noop) in a release line, which is what we want for this?

Reply:
@joyeecheung The difference is that `experimental` usually implies that people should be careful with using it in production, which doesn’t apply here, I think.

Reply:
@addaleax I see. I am tagging this `tsc-review` (I don't think tagging it `tsc-agenda` and discussing it in a meeting  would help?) so we can fully discuss about the status of this flag, just to make sure we don't overlook something before we pushing it out to users..